packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -28,14 +26,39 @@ import triggers from '../helpers/triggers'
 import TriggerLauncher from './TriggerLauncher'
 import VaultCiphersList from './VaultCiphersList'
 import manifest from '../helpers/manifest'
-import { ModalBackButton } from 'cozy-ui/transpiled/react/Modal'
 import HarvestVaultProvider from './HarvestVaultProvider'
 
+import { createOrUpdateCipher } from '../models/cipherUtils'
+
 const IDLE = 'IDLE'
 const RUNNING = 'RUNNING'
 
 const MODAL_PLACE_ID = 'coz-harvest-modal-place'
 
+const buildOrUpdateAccount = ({ account, userData, cipher, konnector }) => {
```

Comment:
Should we move this function to `./connection/accounts` ?

Reply:
I still have some refactoring to do, I will do this later on, is it OK for you ?

Reply:
I have no doubt you will ;)